src/components/layers/VectorLayer.vue

```diff
@@ -274,8 +274,14 @@ function polygonCut(polygon, line, properties) {
       }
     }
 
+    properties['id'] = new Date().getTime()
```

Comment:
What if the ROI already have an id?

Reply:
Id should be a string, not integer, right?

Reply:
resolved, with using randId() func